### PR TITLE
[1.13.x-blue] [KOGITO-7457] - Rebranding changes on the kogito-operator repository

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/main/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains a link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've ran `make before-pr` and everything is working accordingly
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](https://github.com/kiegroup/kogito-operator/blob/main/RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Run operator BDD testing</b>
  Please add comment: <b>/jenkins test</b>

* <b>Run RHPAM operator BDD testing</b>
  Please add comment: <b>/jenkins RHPAM test</b>
</details>

### CODE REVIEW

**Review by sutaakar — APPROVED** [review]

File: Makefile.rhpam [hunk 1]
```diff
@@ -19,20 +19,20 @@ BUNDLE_IMGS ?= $(BUNDLE_IMG)
 CEKIT_CMD := cekit -v --redhat ${cekit_option}
 
 all: generate manifests container-build
-	echo "calling RHPAM all......................................"
+	echo "calling IBM BAMOE all......................................"
```
Comment (radtriste) [inline Makefile.rhpam:22]: Not sure the scripts need update. we just have to update what is customer view.

but the echoes should be harmless :)

File: bundle.Dockerfile [hunk 2]
```diff
@@ -4,9 +4,9 @@ FROM scratch
 LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
-LABEL operators.operatorframework.io.bundle.package.v1=kogito-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=alpha,1.x
-LABEL operators.operatorframework.io.bundle.channel.default.v1=1.x
+LABEL operators.operatorframework.io.bundle.package.v1=rhpam-kogito-operator
```
Comment (radtriste) [inline bundle.Dockerfile:7]: won't this be overriden by https://github.com/kiegroup/kogito-operator/blob/1.13.x-blue/rhpam-image-bundle.yaml ?

File: config/manifests/rhpam/bases/rhpam-kogito-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -7,7 +7,8 @@ metadata:
     categories: Integration & Delivery
     certified: "true"
     containerImage: registry.stage.redhat.io/rhpam-7/rhpam-kogito-rhel8-operator:0.0.0
-    description: RHPAM Kogito Operator for deployment and management of Kogito services.
+    description: IBM BAMOE Kogito Operator for deployment and management of Kogito
+      services.
     operators.openshift.io/valid-subscription: '["Red Hat Process Automation Manager",
```
Comment (radtriste) [inline config/manifests/rhpam/bases/rhpam-kogito-operator.clusterserviceversion.yaml:12]: what will be the subscription ? :S

File: rhpam-image-bundle.yaml [hunk 4]
```diff
@@ -10,11 +10,11 @@ labels:
   - name: com.redhat.component
     value: rhpam-7-kogito-rhel8-operator-bundle-container
   - name: io.k8s.description
-    value: Operator Bundle for deploying RHPAM & RHDM
+    value: Operator Bundle for deploying IBM BAMOE
   - name: io.k8s.display-name
     value: Red Hat Business Automation Operator Bundle
```
Comment (radtriste) [inline rhpam-image-bundle.yaml:15]: To change ?

File: test/pkg/installers/rhpam_kogito_installer.go [hunk 5]
```diff
@@ -52,7 +52,7 @@ var (
 	}
 
 	rhpamKogitoNamespace            = "rhpam-kogito-operator-system"
-	rhpamKogitoServiceName          = "RHPAM Kogito operator"
+	rhpamKogitoServiceName          = "IBM BAMOE Kogito operator"
```
Comment (radtriste) [inline test/pkg/installers/rhpam_kogito_installer.go:55]: I would not change vars to avoid any problem ...

File: Makefile.rhpam (hunk 1 above)
Comment (spolti) [reply Makefile.rhpam:22]: yes :D

File: config/manifests/rhpam/bases/rhpam-kogito-operator.clusterserviceversion.yaml (hunk 3 above)
Comment (spolti) [reply config/manifests/rhpam/bases/rhpam-kogito-operator.clusterserviceversion.yaml:12]: I don't know =/

File: rhpam-image-bundle.yaml (hunk 4 above)
Comment (spolti) [reply rhpam-image-bundle.yaml:15]: yes

File: bundle.Dockerfile (hunk 2 above)
Comment (spolti) [reply bundle.Dockerfile:7]: yes, will revert and update only for release.

**Review by mareknovotny — APPROVED** [review]

File: rhpam-image-bundle.yaml [hunk 6]
```diff
@@ -1,20 +1,20 @@
 schema_version: 1
 name: "rhpam-7/rhpam-kogito-operator-bundle"
-description: "RHPAM Kogito Operator Bundle"
+description: "IBM BAMOE Kogito Operator Bundle"
 version: "7.13.0"
```
Comment (radtriste) [inline rhpam-image-bundle.yaml:4]: Version ?

File: rhpam-image-bundle.yaml [hunk 7]
```diff
@@ -1,20 +1,20 @@
...
 from: "scratch"
 
 labels:
   - name: maintainer
     value: bsig-cloud@redhat.com
   - name: com.redhat.component
-    value: rhpam-7-kogito-rhel8-operator-bundle-container
+    value: ibm-bamoe-7-kogito-rhel8-operator-bundle-container
```
Comment (radtriste) [inline rhpam-image-bundle.yaml:11]: `ibm-bamoe-8` ?

File: rhpam-image.yaml [hunk 8]
```diff
@@ -23,21 +23,21 @@
...
       value: bsig-cloud@redhat.com
-    - name: com.redhat.component
-      value: rhpam-7-kogito-rhel8-operator-container
+    - name: com.ibm.component
+      value: ibm-bamoe-8-kogito-rhel8-operator-container
     - name: io.k8s.description
-      value: Operator for deploying RHPAM Kogito Application
+      value: Operator for deploying IBM BAMOE Kogito Application
     - name: io.k8s.display-name
       value: Red Hat PAM Kogito Operator
```
Comment (davidesalerno) [inline rhpam-image.yaml:38]: If we are changing the description, we should change also the displayed name.

File: config/manifests/rhpam/bases/rhpam-kogito-operator.clusterserviceversion.yaml (hunk 3 above)
Comment (Ginxo) [reply config/manifests/rhpam/bases/rhpam-kogito-operator.clusterserviceversion.yaml:12]: This should be `Business Decision Manager` right?

**Review by radtriste — APPROVED** [review]

**Pull request merged**